vms/rpcchainvm/vm_server.go

```diff
@@ -387,9 +387,8 @@ func (vm *VMServer) CreateStaticHandlers(ctx context.Context, _ *emptypb.Empty)
 func (vm *VMServer) Connected(ctx context.Context, req *vmpb.ConnectedRequest) (*emptypb.Empty, error) {
 	nodeID, err := ids.ToNodeID(req.NodeId)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed notifying connection, nodeID %v: %w", req.NodeId, err)
```

Comment:
This error message seems a bit weird... "notifying connection"... "marking as connected"? Same comment below - I just don't really like "notifying" here